pulsator4droid/src/main/java/pl/bclogic/pulsator4droid/library/PulsatorLayout.java

```diff
@@ -393,13 +438,40 @@ protected void onDraw(Canvas canvas) {
 
     }
 
-    private final Animator.AnimatorListener mAnimatorListener = new Animator.AnimatorListener() {
+    private class AnimatorSimpleListener implements Animator.AnimatorListener {
```

Comment:
Why is this all empty?

Reply:
Because AnimatorListener is an interface, so we want an empty implementation, that we can override for the mAnimationStartListener and mAnimationEndListener. BTW, this was code I cherry-picked from a fix in one of the forks